src/Components/DynamicContent/Config/ConfigManager.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const ConfigManager = (props) => {
+
```

Comment:
Write a //TODO when this is just place holder components.
Would break this component down into smaller components